docker.go

```diff
@@ -252,19 +254,22 @@ func commandBuild(build Build) *exec.Cmd {
 		args = append(args, "--quiet")
 	}
 
-	labelSchema := []string{
-		"schema-version=1.0",
-		fmt.Sprintf("build-date=%s", time.Now().Format(time.RFC3339)),
-		fmt.Sprintf("vcs-ref=%s", build.Name),
-		fmt.Sprintf("vcs-url=%s", build.Remote),
-	}
+	if build.AutoLabel {
+		labelSchema := []string{
+			fmt.Sprintf("created=%s", time.Now().Format(time.RFC3339)),
+			fmt.Sprintf("revision=%s", build.Name),
+			fmt.Sprintf("source=%s", build.Remote),
+			fmt.Sprintf("url=%s", build.Link),
+		}
+		labelPrefix := "org.opencontainers.image"
 
-	if len(build.LabelSchema) > 0 {
-		labelSchema = append(labelSchema, build.LabelSchema...)
```

Comment:
@tboerger can you confirm that `label_schema` values should be ignored if the user sets `auto_label: false`?  This operation is wrapped by `if build.AutoLabel { ...` and I was wondering if it should be moved outside this block, or if this is fine as-is.

Reply:
Adding an option to opt-out is fine, I know some people that really asked for that.